.github/workflows/build.yml

```
@@ -151,8 +151,17 @@ jobs:
         run: echo ::set-output name=version::${GITHUB_REF#refs/*/}
 
       - name: Upload artifact
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ github.event.repository.name }}_${{ steps.version.outputs.version }}_${{ matrix.identifier }}
+          path: |
+            ${{ github.workspace }}/project/
+  
+      - name: Download artifact
+        uses: actions/download-artifact@v4
         with:
           name: ${{ github.event.repository.name }}_${{ steps.version.outputs.version }}
           path: |
             ${{ github.workspace }}/project/
+          pattern: ${{ github.event.repository.name }}*
+          merge-multiple: true
```

```suggestion
        with:
          name: godot-embed-external-editor_${{ steps.version.outputs.version }}_${{ matrix.identifier }}
          path: |
            ${{ github.workspace }}/project/
```

Something like this I think should work